packages/mdc-theme/_variables.scss

```diff
@@ -16,18 +16,43 @@
 
 @import "./functions";
 
-// Main theme colors.
+//
+// Main theme colors for your brand.
+//
 // If you're a user customizing your color scheme in SASS, these are probably the only variables you need to change.
+//
+
 $mdc-theme-primary: #3f51b5 !default; // Indigo 500
+$mdc-theme-primary-light: lighten($mdc-theme-primary, 14%) !default;
+$mdc-theme-primary-dark: darken($mdc-theme-primary, 14%) !default;
+
+// The $mdc-theme-secondary variable is DEPRECATED - it exists purely for backward compatibility.
+// The $mdc-theme-secondary* variables should be used for all new projects.
 $mdc-theme-accent: #ff4081 !default; // Pink A200
+$mdc-theme-secondary: $mdc-theme-accent !default;
```

Comment:
That would break backward compatibility for existing clients who are already overriding `accent`, because we would no longer be reading the value that they're setting.